web-marketplace/src/pages/CreateOrganization/steps/PersonalInfoStep.schema.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from 'zod';
+
+import { TranslatorType } from 'lib/i18n/i18n.types';
+
+import { emailFormSchema } from 'components/organisms/LoginModal/LoginModal.schema';
+
+import { CREATE_ORG_PERSONAL_INFO_NAME_REQUIRED } from '../CreateOrganization.constants';
+
+export const getPersonalInfoSchema = (_: TranslatorType) =>
+  emailFormSchema.extend({
```

Comment:
why using `emailFormSchema`, we can just use `z.string().trim().email()` for the email field instead